README.md

```diff
@@ -1,21 +1,97 @@
+# Compile instruction
+
+## Before compiling
+
+please check the [Releases](https://github.com/Franklalalala/usenauty/releases). There are pre-compiled linux and windows versions, respectively.
+
+## Compile a linux version
+
+The [CXX standard](https://en.wikipedia.org/wiki/C%2B%2B17) is set to be **17**, which means the gcc version need to be 8 or higher. Additionally, the cmake version needs to be **3.1** or higher.
```

Comment:
We use CXX17 because of now is 2023, and some of the dependencies needs new features, but I'm not so sure it is not worked for CXX14 and even CXX11.(Actually I compiled successfully in early version.) So it is not a "must", but a "recommend".